modules/api-svc/src/InboundServer/middlewares.js

```diff
@@ -356,9 +356,10 @@ const createLogger = (logger) => async (ctx, next) => {
         path: ctx.path,
         method: ctx.method
     }});
-    if (!ctx.state.logExcludePaths.includes(ctx.path)) {
-        ctx.state.logger.push({body: ctx.request.body}).log('Request received');
-    }
+    await ctx.state.logger.log('Request received');
+    // if (!ctx.state.logExcludePaths.includes(ctx.path)) {
```

Comment:
I assume we dont have an option to enable debug, etc on the SDK right?

If not, we should add a TODO here, and even possibly re-enable this.